packages/flutter/test/material/text_field_test.dart

```diff
@@ -3382,4 +3382,44 @@ void main() {
     await tester.tap(find.byType(TextField));
     expect(tapCount, 0);
   });
+
+  testWidgets('Includes cursor for TextField', (WidgetTester tester) async {
```

Comment:
We should also verify that if the the TextField is created within an IntrinsicWidth with no stepWidth specified, that the result is a textfield that's just wide enough to drive to the cursor to the end without scrolling. 

Tests use the fixed-width Ahem font by default, so you can easily predict the textfield's exact width.

Should check this with a textfield that's `textAlign: TextAlign.right` and `TextAlign.left`